### PR TITLE
Show api

### PR DESCRIPTION
**Pull request by sethcrist**

### CODE REVIEW

File: src/pages/AddMovie.js [hunk 1]
```diff
@@ -28,8 +28,8 @@ const AddMovie = ({ createList, user, movie }) => {
 				<Row>
 					<Col md={6}>
 						<FormGroup>
-							<Label for="movie-id">{movie.id}</Label>
-							<input
+							<Label for="movie-id">{/*{movie.id}*/}</Label>
```
Comment (sjproctor) [inline src/pages/AddMovie.js:31]: Is this label being used for anything?

File: src/tmdbService.js [hunk 2]
```diff
@@ -1,59 +1,59 @@
-import { useState, useEffect } from "react"
-import axios from "axios"
-import { apiConfig } from "./apiConfig.js"
+import { useState, useEffect } from "react";
```
Comment (sjproctor) [inline src/tmdbService.js]: There are a lot of semicolons in this file that don't exist anywhere else. I would check your linter.

**Review by sjproctor — COMMENTED** [review]
couple small things

File: src/pages/AddMovie.js (hunk 1 above)
Comment (sethcrist) [reply src/pages/AddMovie.js:31]: not yet however it will be

**Review by sjproctor — APPROVED** [review]

**Pull request merged**